### PR TITLE
show VO documentation for brussel

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

**Comment by smoors** [comment]
forgot to run check-version.sh

File: intro-HPC/title.tex [hunk 1]
```diff
@@ -5,7 +5,7 @@
 \vspace*{6\baselineskip}
 
 \Huge \strong{HPC Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: September 25 2020
```
Comment (boegel) [inline intro-HPC/title.tex:8]: You shouldn't commit these dates, the deploy mechanism that creates the PDFs automagically replaces `DATEPLACEHOLDER`

**Review by boegel — CHANGES_REQUESTED** [review]

File: intro-HPC/title.tex (hunk 1 above)
Comment (smoors) [reply intro-HPC/title.tex]: yeah I just saw it.

**Comment by smoors** [comment]
een paar fi-tjes teweining

**Review by boegel — APPROVED** [review]

**Pull request merged**